kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -335,7 +335,7 @@
       backgroundImageStyle() {
         if (this.$theme.signIn.background) {
           const scrimOpacity =
-            this.$theme.signIn.scrimOpacity !== null ? this.$theme.signIn.scrimOpacity : 0.7;
+            this.$theme.signIn.scrimOpacity !== undefined ? this.$theme.signIn.scrimOpacity : 0.7;
```

Comment:
could also use `isNaN` here

Reply:
didn't know about that, but good to know!

Reply:
but also I prefer forcing the user to pass in a specific value, hence the opacity `None` checks